aten/src/ATen/core/dispatch/DispatchTable.h

```diff
@@ -291,25 +291,11 @@ class DispatchTable final {
               "Available functions are ", listAllDispatchKeys())
       }
 
-      // If the input is quantized, but the quantization is not supported.
-      if (dispatch_key.value() == TensorTypeId::QuantizedCPUTensorId) {
-        TORCH_CHECK(false, "Tried running '", operator_name_, "' with a",
-                    " quantized tensor but '", operator_name_, "' expects a",
-                    " non-quantized input.");
-      }
-
-      // If the input is not quantized, but the kernel is.
-      if (kernels_.lookup(TensorTypeId::QuantizedCPUTensorId)) {
-        TORCH_CHECK(false, "Tried running '", operator_name_, "' but the input",
-                    " is not quantized. Please ensure you have QuantStub",
-                    " during model conversion, or you manually quantize the",
-                    " input tensor.");
-      }
-
       const std::string dispatch_key_str = toString(*dispatch_key);
-      TORCH_CHECK(false, "Didn't find kernel to dispatch to for operator '", operator_name_,
-               "'. Tried to look up kernel for dispatch key '", dispatch_key_str,
-               "'. Registered dispatch keys are: ", listAllDispatchKeys());
+      TORCH_CHECK(false, "Could not run '", operator_name_, "' with its input",
```

Comment:
`with arguments from the ... backend` is probably better than `input` because python developers think in functions not in graph nodes with input connections.